src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -36,6 +23,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.KeyPair;
+import java.util.*;
```

Comment:
nit: unsure why these imports were changed?

Reply:
+1 no wildcard import. Probably some intellij magic.

Reply:
will update, i update one intellJ so all the settings seems get lost